src/runtime_config/resolve_veto.js

```diff
@@ -0,0 +1,27 @@
+export function resolve(proposal, proposerId, votes) {
+    let memberVoteCount = 0;
+    for (const vote of votes) {
+        if (vote.vote) {
```

Comment:
Looking at this example, I think this field should be `vote.in_favour`, or `vote.for`, or `vote.supports`, or something. To avoid the duplicated `vote.vote`, and so we can more clearly distinguish "did you submit ballot executed to completion" from "do you support this proposal".

Reply:
Agreed, although `for` is probably not going to work.